### ISSUE TITLE
Release v2.7.0

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/openfoodnetwork#4679]
Steps: 

- [x] Include translations
- [x] Draft: https://github.com/openfoodfoundation/openfoodnetwork/releases/edit/untagged-e6aef06f2b5af6ccda58
- [x] Test: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master/builds/1106
- [x] Publish and notify #global-community
- [x] Deploy and notify #instance-managers
- [x] Nudge next release manager

The full process is described at https://github.com/openfoodfoundation/openfoodnetwork/wiki/Releasing.

**Comment by lin-d-hop** [comment]
Testing the release on FR staging:
I opened a shop (Marsh Goods) with Collection and Delivery, Paypal, Stripe, Cash, two OCs
Added products (Insects, Weeds) to the cart. 
When I click "checkout" I get a 500 error

Release testing on hold until a dev can take a look. Thank you please :-)

**Comment by lin-d-hop** [comment]
https://github.com/openfoodfoundation/openfoodnetwork/issues/4689

**Comment by lin-d-hop** [comment]
Ok well this is a bug on many fronts.
1) Even when I removed Stripe as a payment method the problem persists.
2) Everything is connected fine in the backend.

So it could be related to one of two different things:

a) #4606 - somehow this hasn't worked. I used an existing Stripe account and connected to this account. Does the fix in some way assume that this payment method will not be connected to other enterprises?

b) Doesn't look like any of the Stripe upgrades are in this release now that I check....  so that rules out my second thing :-)
